crml/cennzx-spot/src/lib.rs

```diff
@@ -332,10 +317,9 @@ decl_storage! {
 		pub CoreAssetId get(core_asset_id) config(): T::AssetId;
 		/// Default Trading fee rate
 		pub DefaultFeeRate get(fee_rate) config(): FeeRate<PerMillion>;
-		/// Total supply of exchange token in existence.
+		/// Total liquidity in each individual exchange.
```

Comment:
Total liquidity of all investors in an exchange
i.e. total_liquidity(exchange) == sum(liquidity_balance(exchange, user)) at all times

^or something to that effect